linkerd-tcp/linkerd-tcp.yml

```diff
@@ -0,0 +1,13 @@
+admin:
```

Comment:
mind annotating this file a bit with comments? particularly useful since it's early days.